src/uct/ib/base/ib_iface.c

```diff
@@ -276,17 +281,20 @@ void uct_ib_address_pack(const union ibv_gid *gid, uint16_t lid,
         *(uint16_t*) ptr = lid;
```

Comment:
maybe remove `UCT_IB_ADDRESS_FLAG_LINK_LAYER_IB` and `UCT_IB_ADDRESS_FLAG_LID` flags, as they look excessive now?

Reply:
this will be done as part of #4688